TI_Practice1/Practice1.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace Practice1
+{
+    class Practice1
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("***Practice1: Start***");
+
+            int attempts = 3;
+            double radius;
+            double side;
+            double result;
+
+            Console.Write($"\nPlease enter a number used as a circle radius. You have {attempts} attempts: ");
+
+            Validator validateRadius = new Validator(attempts);
+            radius = validateRadius.IsNumber();
+
+            Console.Write($"\nPlease enter a number used as a square side. You have {attempts} attempts: ");
+
+            Validator validateSide = new Validator(attempts);
+            side = validateSide.IsNumber();
+
+            Circle circle = new Circle(radius);
+            result = Math.Round(circle.GetArea(), 2);
+            Console.Write("\n\nCircle area: " + result);
+
+            Square square = new Square(side);
+            result = Math.Round(square.GetArea(), 2);
+            Console.Write("\nSquare area: " + result);
+
+            if (circle.GetDiametr() >= square.GetDiagonal())
+            {
+                Console.Write("\nSquare can be put inside circle.");
+            }
+            else
+            {
+                Console.Write("\nCircle can be put inside square.");
+            }
```

Comment:
I believe this solution is not correct.
It says "Circle can be put inside square." for the circle radius = 4 and square side = 6.

![image](https://user-images.githubusercontent.com/9382934/109701636-e6f69880-7b9b-11eb-84c2-8131315b56c2.png)

Could you please review the solution?

Reply:
DONE. My faill

Reply:
DONE My fail